include/libtorrent/kademlia/node.hpp

```diff
@@ -134,7 +134,7 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	int data_size() const { return int(m_storage.num_torrents()); }
 #endif
 
-#ifndef TORRENT_DISABLE_LOGGING
+#if TORRENT_USE_IOSTREAM
 	// TODO: 3 make this print to the DHT logger instead
```

Comment:
I thought about it, but I don't think this `print_state` is a good fit for the logging, it seems like a potentially very big string, that's the reason I didn't tried to move it to the log.